### DISCUSSION TITLE
Add support for robotframework

### DISCUSSION THREAD

**Discussion by nmathew** [LazyVim/LazyVim#3995] [Extra Requests]
May be we should add support for robot.

```
return {
  recommended = function()
    return LazyVim.extras.wants({
      ft = { "robot" },
      root = { "robotidy.toml", "pyproject.toml", "conda.yaml", "robot.yaml" },
    })
  end,
  {
    "neovim/nvim-lspconfig",
    opts = {
      servers = {
        robotframework_ls = {},
      },
    },
  },
  {
    "nvim-treesitter/nvim-treesitter",
    opts = { ensure_installed = { "robot" } },
  },
}
```

**Comment by lkoning** [comment]
I would love to have a LazyExtra package to get going with Robotframework fast.

There is this RobotCode project also having a lsp implementation, with complementing Robocop and Robotidy for linting/formatting. The biggest challenge is that all these tools usually are installed in a users venv of a project. 

I haven tried everything, but I haven't got it to play nice with Nvim yet.

See also: https://github.com/mason-org/mason-registry/pull/4886
